ntoskrnl/io/pnpmgr/pnpres.c

```diff
@@ -12,6 +12,16 @@
 #define NDEBUG
```

Comment:
https://jira.reactos.org/browse/CORE-10146 contains some info about that resource list parsing problem.

Reply:
Yeah, I added another hack to make it work a bit better, but it's still a bug. We need to figure out why these conflicts exist and fix it.